pkgs/development/python-modules/asyncua/default.nix

```diff
@@ -65,9 +61,10 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "asyncua" ];
 
-  disabledTests = lib.optionals stdenv.isDarwin [
+  disabledTests = [
     # Failed: DID NOT RAISE <class 'asyncio.exceptions.TimeoutError'>
     "test_publish"
+  ] ++ lib.optionals stdenv.isDarwin [
     # OSError: [Errno 48] error while attempting to bind on address ('127.0.0.1',...
```

Comment:
Try `__darwinAllowLocalNetworking = true;`

Reply:
I can't test on Darwin and I doubt @ofborg will ever build it.

Reply:
Ah, `error while attempting to bind on address ('0.0.0.0'`, tests try to bind to 0.0.0.0, not to 127.0.0.1 as stated in comment, so __darwinAllowLocalNetworking doesn't help here.